src/main/java/core/basesyntax/strategy/Strategy.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface Strategy {
+    void getOperation(FruitTransaction operation);
```

Comment:
void method name should not begin with "get". We can not **get** anything from void method